shared/base/view.js

```diff
@@ -258,6 +258,15 @@ module.exports = BaseView = Backbone.View.extend({
       };
     }
 
+    // Allow ability to just pass the full "spec" to a lazy loaded view
+    if (this.options.fetch_spec) {
```

Comment:
The variable will be coming from the template when using rendr-handlebars which is case insensitive.